awx/ui/src/screens/NotificationTemplate/shared/TypeInputsSubForm.js

```diff
@@ -362,8 +362,14 @@ function SlackFields() {
         type="textarea"
         validate={required(null)}
         isRequired
-        tooltip={t`Enter one Slack channel per line. The pound symbol (#)
-          is required for channels.`}
+        tooltip={
+          <>
+            {t`Enter one Slack channel per line. The pound symbol (#)
+          is required for channels. To respond to or start a thread to a specific message add the parent message Id to the channel where the parent message Id is 16 digits. A dot (.) must be manually inserted after the 10th digit.  ie:#destination-channel, 1231257890.006423. See Slack`}{' '}
```

Comment:
The example here has a space after the comma. Eyeballing the python code that parses this it would result in `thread = ' 1231257890.006423'` will slack handle the space OK ? Easy fix would be to change this example. Harder fix would be to strip whitespace at parse time in python.

Reply:
The notifications send successfully with or without a space after the comma, so this tooltip example is :+1: